WeakAuras/Init.lua

```diff
@@ -368,6 +368,21 @@ local flavorFromTocToNumber = {
 }
 local flavor = flavorFromTocToNumber[flavorFromToc]
 
+
```

Comment:
so the order is deliberate here, WeakAuras.buildType is repeatedly rewritten to progressively less stable flavors until it's correct. Old code tried to have the line order of this reversed, which required a more complex house of cards. I could have fixed that but it would have required thinking about regular expressions and I didn't feel like doing that.